hedy.py

```diff
@@ -1093,10 +1102,10 @@ def ask(self, args):
 
         tryblock = textwrap.dedent(f"""
         try:
-          prijs = int({var})
+          {var} = int({var})
```

Comment:
was `prijs` eerder verkeerd?

Reply:
Yes that was a copy-paste mistake (oops)